DESCRIPTION

```diff
@@ -29,6 +29,7 @@ Imports:
     gh,
     git2r (>= 0.23),
     glue,
+    purrr,
```

Comment:
OK then!

Reply:
It's now light weight, and makes life so much eaiser.